src/url-params/url-params.js

```diff
@@ -169,13 +219,8 @@
 
```

Comment:
any reason you don't want to put the payload and validation in the update url with tag method? maybe make it a updateurlwithcomponentpayload or something?